pkg/synopsysctl/helpers.go

```diff
@@ -46,6 +46,6 @@ func addbaseURLFlag(cmd *cobra.Command) {
 }
 
 func addChartLocationPathFlag(cmd *cobra.Command) {
-	cmd.Flags().StringVarP(&baseURL, "chart-location-path", "", baseURL, "Absolute path to the Helm Chart Tarball")
-	cmd.Flags().MarkHidden("chart-location-path")
+	cmd.Flags().StringVarP(&baseURL, "chart-location-path", "", baseURL, "Absolute path to the Helm Chart Tarball for air-gapped customer")
```

Comment:
Should we say Helm Chart Tarball? Or just "Application Tarball" or "Resource Tarball"?